### DISCUSSION TITLE
New updates upstream

### DISCUSSION THREAD

**Discussion by Zhincore** [anapnoe/stable-diffusion-webui-ux#186] [Q&A]
Hello, me again.
I've heard A1111 had some big updates, are we gonna get them too?
Will Vladmandic get UX update instead?

Thanks

**Comment by Neverdusk** [comment]
Does anyone know if this branch still being developed? Or if there's a fork of this branch with Auto's 1.6 update?

**Reply by Zhincore** [reply]
There is dev-1.6 branch but that one is not usable
